### PR TITLE
[GEOS-8321] Monitoring plugin Path Exception

### PR DESCRIPTION
**Pull request by afabiani**

### CODE REVIEW

File: src/extension/monitor/core/src/main/java/org/geoserver/monitor/rest/RESTMonitorCallback.java [hunk 1]
```diff
@@ -43,9 +46,14 @@ public void init(HttpServletRequest request, HttpServletResponse response) {
         
         data.setCategory(Category.REST);
         if (request.getPathInfo() != null) {
-            String resource = Paths.get(request.getPathInfo()).getFileName().toString();
-            resource = FilenameUtils.getBaseName(resource);
-            data.getResources().add(resource);
+        	try {
```
Comment (aaime) [inline src/extension/monitor/core/src/main/java/org/geoserver/monitor/rest/RESTMonitorCallback.java]: Needs a test (like any other bug fix).

Comment (afabiani) [reply src/extension/monitor/core/src/main/java/org/geoserver/monitor/rest/RESTMonitorCallback.java]: Added.

**Comment by afabiani** [comment]
@aaime done.

**Comment by aaime** [comment]
Looks ok. I was wondering about the "lenght - 1", but cannot think of a case in which the result of split would be zero length.

**Pull request merged**